From 38d3bea29a7c19a5f41d347b7658fa058c56a942 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sun, 2 Nov 2014 10:02:41 -0500
Subject: [PATCH] Revert "units: define appropriate job timeout actions when
 boot or shutdown timeouts are hit"

This reverts commit 3898b80d409ae16b049d46f883bf763417bb4c8a.

We have various jobs that run during early boot and might cause
basic.target not to be reached within 15 minutes. E.g. Fedora has
fedora-autorelabel.service, which can take an arbitrarily long time on
a large and slow drive.
---
 units/basic.target    | 2 --
 units/poweroff.target | 2 --
 units/reboot.target   | 2 --
 3 files changed, 6 deletions(-)

diff --git a/units/basic.target b/units/basic.target
index 228f62c4b1..b890d48bbc 100644
--- a/units/basic.target
+++ b/units/basic.target
@@ -11,5 +11,3 @@ Documentation=man:systemd.special(7)
 Requires=sysinit.target
 Wants=sockets.target timers.target paths.target slices.target
 After=sysinit.target sockets.target timers.target paths.target slices.target
-JobTimeoutSec=15min
-JobTimeoutAction=poweroff-force
diff --git a/units/poweroff.target b/units/poweroff.target
index dd92d816ca..71871033a5 100644
--- a/units/poweroff.target
+++ b/units/poweroff.target
@@ -12,8 +12,6 @@ DefaultDependencies=no
 Requires=systemd-poweroff.service
 After=systemd-poweroff.service
 AllowIsolate=yes
-JobTimeoutSec=30min
-JobTimeoutAction=poweroff-force
 
 [Install]
 Alias=ctrl-alt-del.target
diff --git a/units/reboot.target b/units/reboot.target
index 668b98d9e4..dec8f56796 100644
--- a/units/reboot.target
+++ b/units/reboot.target
@@ -12,8 +12,6 @@ DefaultDependencies=no
 Requires=systemd-reboot.service
 After=systemd-reboot.service
 AllowIsolate=yes
-JobTimeoutSec=30min
-JobTimeoutAction=reboot-force
 
 [Install]
 Alias=ctrl-alt-del.target
